.github/ISSUE_TEMPLATE/bug-en.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug
+about: Create a bug report to help us improve
+title: "[BUG] Feature name - Short description of the bug"
+labels: ''
+assignees: ''
+
+---
+
+## Required information
+
+**Describe the bug**
```

Comment:
```suggestion
**Describe the bug**
<!-- Mandatory -->
```

Reply:
Having it in the preceding section header is sufficient.